content/concepts/introduction/technical-faq.md

```diff
@@ -0,0 +1,99 @@
+
+No. The Ripple network is opt-in. Each participant directly or indirectly chooses its UNL. Should _Ripple_ stop operating or should _Ripple_ act maliciously, participants could change their UNLs to continue using the network.
+
+
+#### What is the validator incentive structure for validators not run by _Ripple_?
+
+If the Ripple network becomes successful and is widely used for interbank settlement, there will be an incentive for participants to ensure the reliability and stability of the network. If this happens, institutions will run Ripple servers to participate in the network. Once you are running a server, the additional cost and effort to operate a validator is essentially zero—it would simply involve flipping a software switch from off to on. It is the validators who decide the evolution of the Ripple network, so the primary incentive to run a validator is to preserve and protect the stable operation and sensible evolution of the network.
+
+
+#### Can financial institutions set up transaction validators that will help them meet specific institutional standards and requirements?
+
+No, institutions cannot set up customized validator policies for transaction selection. Validators either follow the protocol, or they do not. If software does not follow protocol rules, it will not function. Thus, it is not recommended that institutions seek out custom implementations without in-house expertise.
+
+
+#### What will happen if more than 20% of nodes within the network do not agree with the majority? How is the final version of the ledger chosen?
+
+The network may temporarily halt to reconfigure itself to continue with the new UNL list based on those that want to reach consensus. This temporary processing delay is desired rather than double spending.
+
+In the process of determining the final, authoritative version of the ledger, there may be multiple temporary internal versions. Such internal versions  will happen in distributed systems because not all nodes will receive transactions in the same order. The analogous behavior in Bitcoin is where two servers each see a different longest chain because two blocks were mined at about the same time.
+
+However, there will only be one authoritative ledger version at any given time; other versions are irrelevant and harmless.
+
+
+#### Does Ripple utilize a formal validator onboarding process?
+
+No, a formal validator onboarding process is not compatible with Ripple, as it is a system with no central authority. Rather, _Ripple_ provides recommendations and best practices.
+
+
+## Role of XRP
+
+
+#### Why does _Ripple_ utilize XRP holdings?
+
+_Ripple_ XRP holdings incentivize the company to make Ripple as useful as possible. XRP exists as a native asset on Ripple for anti-spam transaction purposes, and for currency bridging only if beneficial to users. Otherwise, the use of XRP in transactions is completely optional.
```

Comment:
"_Ripple_ XRP holdings" → "Ripple's XRP holdings"

"to make Ripple as useful as possible" → "to make the XRP Ledger" as useful as possible"

"a native asset on Ripple" → "a native asset in the XRP Ledger"